torchvision/models/resnet.py

```diff
@@ -112,11 +112,10 @@ def __init__(self, block, layers, num_classes=1000):
 
         for m in self.modules():
             if isinstance(m, nn.Conv2d):
-                n = m.kernel_size[0] * m.kernel_size[1] * m.out_channels
-                m.weight.data.normal_(0, math.sqrt(2. / n))
+                nn.init.kaiming_normal_(m.weight, mode="fan_out")
```

Comment:
I still think you need to pass the `nonlinearity='relu'` here, if not we are not computing the same thing

Reply:
Looking again, it might be the case that this doesn't matter here because `a=0` by default, so we would have maybe the same result. Still, might be good to change just to be safe